src/include/common/enums/alter_type.h

```diff
@@ -11,6 +11,7 @@ enum class AlterType : uint8_t {
     ADD_PROPERTY = 10,
     DROP_PROPERTY = 11,
     RENAME_PROPERTY = 12,
+    COMMENT = 13,
```

Comment:
make this 20. We want to leave gaps between different categories of AlterType, e.g. ADD/DROP/RENAME_PROPERTY have consecutive uint8_t, and if the future if we add `XXX_PROPERTY` we wanna make it 13.